.github/workflows/PR-checker.yml

```diff
@@ -0,0 +1,15 @@
+name: production build
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Wider scope? This runs ONLY for master and nothing else, it's not a PR check anymore.

Reply:
It runs on pull_request to master